apps/charterafrica/src/components/EventsFellowshipsGrantsHeader/EventsFellowshipsGrantsHeader.js

```diff
@@ -4,8 +4,8 @@ import React from "react";
 
 import { secondary } from "@/charterafrica/colors";
 
-const GrantsFellowshipsHeader = React.forwardRef(
-  function GrantsFellowshipsHeader(props, ref) {
+const EventsFellowshipsGrantsHeader = React.forwardRef(
```

Comment:
```suggestion
const FundingHeader = React.forwardRef(
```

Or `OpportunityHeader` should be enough